internal/cmd/server/log/log_test.go

```diff
@@ -180,7 +180,7 @@ func TestOutputResult(t *testing.T) {
 	type args struct {
 		outputFormat string
 		serverLabel  string
-		log          string
+		logs         []string
```

Comment:
```suggestion
		logLines         []string
```

Please use the param names from the func you're testing if possible